### DISCUSSION TITLE
Dark Mode

### DISCUSSION THREAD

**Discussion by MrMK0R** [uxbox/uxbox#3792] [Ideas]
Lots of design software has made the swap over to dark mode (web apps like Figma, GitHub, and desktop apps like affinity designer).

It's a lot easier on the eyes and would help if it was just a quick switch (like hovering in the corner).

This would include the login, main screen, and inside the design file (making the base canvas light grey and the canvas background dark grey.

**Comment by madalenapmelo-kp** [comment]
Hi @MrMK0R,

Thanks for the suggestion! We're already developing this and it will be released soon, you can stay up to date on the progress [here](https://tree.taiga.io/project/penpot/us/2287) or by keeping an eye on issues #1769 or #1590.

I'm closing this discussion but please feel free to share any other suggestions on [our Community](https://community.penpot.app/) as this forum has been discontinued. Thanks :slightly_smiling_face:

**Comment by evolve2k** [comment]
Checking in on Dark Mode as the links out seem to say its closed/resolved? but I'm not sure I've seen the feature live.
Could you share an update or take this as a gentle poke that folks still want this feature!

**Reply by madalenapmelo-kp** [reply]
This feature is live, you can switch now between Dark and Light mode: https://help.penpot.app/user-guide/the-interface/#interface-ui-theme :slightly_smiling_face:

**Reply by evolve2k** [reply]
Awesome! Thankyou !!!!
